app/src/app/utils/events/handlers.ts

```diff
@@ -11,8 +14,19 @@ import { MapStore } from "@/app/store/mapStore";
  */
```

Comment:
we can surely do better than this, but is a first stab. a lot will depend on how we format data coming from the server to look / where it will be stored

Reply:
uhoh- there is a sync issue! it seems if blocks are taken from one zone to another, the pop sum of the sender is not reduced properly (though the receiver is)